### DISCUSSION TITLE
Having Bonsai communicate with arduino to send TTL

### DISCUSSION THREAD

**Discussion by albitcaban** [bonsai-rx/bonsai#1457] [Q&A]
Hi, I'm looking to time-lock a camera with my miniscope data. To do this I want to have Bonsai send my miniscope DAQ a constant 5v signal as soon as I click start on Bonsai, this will in turn tell my DAQ (through a BNC cable attached to PIN 2)  to start the calcium recording on the miniscope while simultaneously starting my video camera (for behavior) recording on Bonsai. I have tried some nodes so far but it does not seem to work. I also uploaded Firmata to my arduino. I'm very confused with the documentation regarding bonsai to arduino connections and need a little help. Please let me know what I might be doing wrong or if anyone has any ideas on how to organize nodes and which nodes to use accordingly! Thank you!

**Comment by bruno-f-cruz** [comment]
Heya,
This is still an [ongoing article](https://github.com/bruno-f-cruz/docs-wip/blob/%2320/articles/ArduinoArticle.md), but check if it helps:
